client/my-sites/plans-v2/plans-column/index.tsx

```diff
@@ -102,6 +102,25 @@ const PlansColumn = ( { duration, onPlanClick, productType, siteId }: PlanColumn
 				...product,
 				owned: product.productSlug === currentPlan,
 				legacy: false,
+			} ) )
+			// If product offers 'realtime' and 'daily' options, AND product is not already owned,
+			// append "Available Options: Real-Time and Daily" to the product description.
+			.map( ( product ) => ( {
+				...product,
+				description:
+					product.subtypes?.filter( ( subtype ) => /_(daily|realtime)$/.test( subtype ) ).length >=
```

Comment:
This regex needs to be updated because it would not work with monthly plans (e.g. `jetpack_security_daily_monthly`). Alternatively, you could try to get the plan object by its slug (`subtype` here) and check if an attribute indicates whether it's daily or realtime, but I don't know if such an attribute exists.

Reply:
Just an idea. What if we add a utility function that could answer the question by checking the `subtype` against a two lists of product slugs? I imagine one list could contain Daily products and the other Real-time products.

Reply:
Seems more scalable to me indeed.

Reply:
Ok, yes sounds good!  I've refactored it into a single function that compares against two lists of Daily and Real-time products. It should be much better now. :)

** Also I've since the PR got so far behind master (and there were a bunch of conflicts to resolve before it could be tested), [I moved the changes over to this new updated PR](https://github.com/Automattic/wp-calypso/pull/45197)